### DISCUSSION TITLE
Which brokers supports delay and priority?

### DISCUSSION THREAD

**Discussion by merura** [taskiq-python/taskiq#189] [Q&A] [answered]
Which brokers have implemented delay and priority features mentioned in [documentation](https://taskiq-python.github.io/extending-taskiq/broker.html#conventions)?
I briefly looked into repositories of some Brokers and found out that only [aio-pika broker](https://github.com/taskiq-python/taskiq-aio-pika) supports them. I would like to have list of supported brokers in documentation

**Comment by s3rius** [comment, marked as answer]
Currently only aio-pika broker supports these features. Since other brokers don't have such features out of the box. 

If you want delays, you can easily create dynamic schedule source. As shown here: https://github.com/taskiq-python/taskiq/issues/187.
